core/src/main/java/org/apache/iceberg/rest/requests/UpdateTableRequest.java

```diff
@@ -95,6 +117,12 @@ public Builder(TableMetadata base, boolean isReplace) {
       this.isReplace = isReplace;
     }
 
+    public Builder forTable(TableIdentifier ident) {
```

Comment:
I don't think we need this if we are replacing the `UpdateTableRequest` right?

Reply:
yep, I removed this already in #7741